From 9be3539bf4c73c786b2d479bb4dccb04e270232f Mon Sep 17 00:00:00 2001
From: mohammadsohail17 <sohail23264@gmail.com>
Date: Sun, 23 Oct 2022 11:37:15 +0530
Subject: [PATCH] client.py modified

---
 client.py | 8 +++++---
 1 file changed, 5 insertions(+), 3 deletions(-)

diff --git a/client.py b/client.py
index 3fc09b7..1ce8dd5 100644
--- a/client.py
+++ b/client.py
@@ -35,14 +35,16 @@ def getDataPoint(quote):
     stock = quote['stock']
     bid_price = float(quote['top_bid']['price'])
     ask_price = float(quote['top_ask']['price'])
-    price = bid_price
+    price = (ask_price+bid_price)/2
     return stock, bid_price, ask_price, price
 
 
 def getRatio(price_a, price_b):
     """ Get ratio of price_a and price_b """
     """ ------------- Update this function ------------- """
-    return 1
+    if (price_b==0):
+        return 
+    return price_a/price_b
 
 
 # Main
@@ -56,4 +58,4 @@ if __name__ == "__main__":
             stock, bid_price, ask_price, price = getDataPoint(quote)
             print("Quoted %s at (bid:%s, ask:%s, price:%s)" % (stock, bid_price, ask_price, price))
 
-        print("Ratio %s" % getRatio(price, price))
+        print("Ratio %s" % getRatio(price['ABC'], price['DEF']))
-- 
2.33.1.windows.1

